radix-engine-interface/src/blueprints/resource/access_rules.rs

```diff
@@ -191,13 +191,13 @@ impl AuthorityRules {
         );
     }
 
-    pub fn set_owner_rule(&mut self, rule: AccessRule, mutability: AccessRule) {
+    pub fn set_owner_authority(&mut self, rule: AccessRule, mutability: AccessRule) {
         self.rules.insert(AuthorityKey::Owner, (rule, mutability));
     }
 
-    pub fn owner_authority(owner_badge: &NonFungibleGlobalId) -> AuthorityRules {
+    pub fn create_owner_authority(owner_badge: &NonFungibleGlobalId) -> AuthorityRules {
         let mut authority_rules = AuthorityRules::new();
-        authority_rules.set_owner_rule(
+        authority_rules.set_owner_authority(
             rule!(require(owner_badge.clone())),
             rule!(require(owner_badge.clone())),
```

Comment:
It's cheaper in terms of space to set the mutability of the owner authority to be the owner authority.

Reply:
Ah yes, this is interesting